doc/classes/TileMapLayer.xml

```diff
@@ -0,0 +1,308 @@
+			</description>
+		</method>
+		<method name="update_internals">
+			<return type="void" />
+			<description>
+				Triggers a direct update of the TileMapLayer. Usually, calling this function is not needed, as TileMapLayer node updates automatically when one of its properties or cells is modified.
+				However, for performance reasons, those updates are batched and delayed to the end of the frame. Calling this function will force the TileMapLayer to update right away instead.
+				[b]Warning:[/b] Updating the TileMapLayer is computationally expensive and may impact performance. Try to limit the number of updates and how many tiles they impact.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="collision_enabled" type="bool" setter="set_collision_enabled" getter="is_collision_enabled" default="true">
+			Enable or disable collisions.
+		</member>
+		<member name="collision_visibility_mode" type="int" setter="set_collision_visibility_mode" getter="get_collision_visibility_mode" enum="TileMapLayer.VisibilityMode" default="0">
+			Show or hide the TileMapLayer's collision shapes. If set to [constant VISIBILITY_MODE_DEFAULT], this depends on the show collision debug settings.
+		</member>
+		<member name="enabled" type="bool" setter="set_enabled" getter="is_enabled" default="true">
+			If [code]false[/code], disables completely this TileMapLayer (rendering, collision, navigation, scene tiles, etc.)
+		</member>
+		<member name="navigation_enabled" type="bool" setter="set_navigation_enabled" getter="is_navigation_enabled" default="true">
+			Enable or disable navigation regions.
+		</member>
+		<member name="navigation_visibility_mode" type="int" setter="set_navigation_visibility_mode" getter="get_navigation_visibility_mode" enum="TileMapLayer.VisibilityMode" default="0">
+			Show or hide the TileMapLayer's navigation meshes. If set to [constant VISIBILITY_MODE_DEFAULT], this depends on the show navigation debug settings.
+		</member>
+		<member name="rendering_quadrant_size" type="int" setter="set_rendering_quadrant_size" getter="get_rendering_quadrant_size" default="16">
+			The TileMapLayer's quadrant size. A quadrant is a group of tiles to be drawn together on a single canvas item, for optimization purposes. [member rendering_quadrant_size] defines the length of a square's side, in the map's coordinate system, that forms the quadrant. Thus, the default quandrant size groups together [code]16 * 16 = 256[/code] tiles.
+			The quadrant size does not apply on a Y-sorted TileMapLayer, as tiles are be grouped by Y position instead in that case.
+			[b]Note:[/b] As quadrants are created according to the map's coordinate system, the quadrant's "square shape" might not look like square in the TileMapLayer's local coordinate system.
+		</member>
+		<member name="tile_map_data" type="PackedByteArray" setter="set_tile_map_data_from_array" getter="get_tile_map_data_as_array" default="PackedByteArray(0, 0)">
```

Comment:
This wasn't exposed before.

Reply:
It is exposed in TileMap. It is just not documented as it is exposed via `get`/`set`/`get_property_list`.
Now I could expose it as a property, so I had to expose a `getter` and a `setter` too.

But in any case, I believe exposing it is good anyway.

Reply:
So long as the Inspector doesn't show it, it should be fine. Although I can't recall a built-in property that does _not_ show in the Inspector. To accomplish this I've only seen  their setters and getters (e.g. `set_process()`), but that may be because these methods are old.